packages/advanced-logic/src/extensions/payment-network/bitcoin/testnet-address-based.ts

```diff
@@ -13,23 +11,10 @@ const BITCOIN_NETWORK = 'testnet';
  * Every bitcoin transaction that reaches these addresses will be interpreted as payment or refund.
  * Important: the addresses must be exclusive to the request
  */
-export default class BitcoinTestnetAddressBasedPaymentNetwork extends AddressBasedPaymentNetwork {
+export default class BitcoinTestnetAddressBasedPaymentNetwork extends BitcoinAddressBasedPaymentNetwork {
```

Comment:
To make `codeclimate` happy, `BitcoinTestnetAddressBasedPaymentNetwork` now extends `BitcoinAddressBasedPaymentNetwork` :) Which isn't a bad idea IMO

Reply:
looks reasonable indeed !